content/docs/0.8.0/users-guide/default-disk-and-node-config.md

```diff
@@ -1,41 +1,78 @@
 
-This feature is available since v0.8.1.
+Customizing the default configuations for disks and nodes is useful for scaling the cluster because it eliminates the need to configure Longhorn manually for each new node if the node contains more than one disk, or if the disk configuration is different for new nodes.
 
-## Usage
+> **Note:** Longhorn will not keep the node labels or annotations in sync with the current Longhorn node disks or tags.
 
-### Add node tags for new nodes
+### Adding Node Tags to New Nodes
 
-1. Scale up the Kubernetes cluster. Then the newly added nodes contain no node tag.
-2. Add the annotations to the new Kubernetes nodes. 
-    - The annotation format is `node.longhorn.io/default-node-tags: <node tag list with JSON string format>`. For example:
+When a node does not have a tag, Longhorn uses a node annotation to set the node tags.
+
+1. Scale up the Kubernetes cluster. The newly added nodes contain no node tags.
+2. Add annotations to the new Kubernetes nodes that specify what the default node tags should be. The annotation format is:
+
+    ```
+    node.longhorn.io/default-node-tags: <node tag list with JSON string format>
+    ```
+    For example:
+    
     ```
     node.longhorn.io/default-node-tags: '["fast","storage"]'
     ``` 
-3. Wait for Longhorn syncing the node tag automatically.
+3. Wait for Longhorn to sync the node tag automatically.
+
+> **Result:** If the node tag list was originally empty, Longhorn updates the node with the tag list, and you will see the tags for that node updated according to the annotation. If the node already had tags, you will see no change to the tag list.
+
+### Customizing Default Disks for New Nodes
+
+Longhorn uses the **Create Default Disk on Labeled Nodes** setting to enable default disk customization.
+
+- If the setting is enabled, Longhorn waits for the default disk customization to be set.
+- If the setting is disabled, Longhorn creates the default disk for new nodes that don't have existing disks. Then Longhorn relies on the value of the node label node.longhorn.io/create-default-disk to decide how to customize the default disks.
+
+If the value of the label is `config`, the annotation will be parsed and used as the default disk customization. If the value is the boolean value `true`, the data path setting will be used for the default disk. Other values will be treated as false and no default disk will be created.
+
+The default disk will not be automatically created on the new nodes unless the `create-default-disk` label is set.
 
-### Customize default disks for new nodes
+The configuration described in the annotation only takes effect when there are no existing disks or tags on the node.
+
+If the label or annotation fails validation, the whole annotation is ignored. 
+
+> **Prerequisite:** The Longhorn setting **Create Default Disk on Labeled Nodes** must be enabled.
+
+1. Add new nodes to the Kubernetes cluster, for example, by using Rancher or Terraform.
```

Comment:
Don't need the `for example` part.

Reply:
> 1. Add new nodes to the Kubernetes cluster. Rancher or Terraform can be used to add nodes.

Reply:
I meant, just `Add new nodes to the Kubernetes cluster.` would be enough.

Reply:
updated